packages/lodestar/src/eth1/eth1MergeBlockTracker.ts

```diff
@@ -142,8 +143,12 @@ export class Eth1MergeBlockTracker {
     this.close();
   }
 
-  private startFinding(): void {
+  private async startFinding(): Promise<void> {
     if (this.status !== StatusCode.PRE_MERGE) return;
+    const powBlockOverride = await this.getPowBlock(toHexString(this.config.TERMINAL_BLOCK_HASH));
```

Comment:
This getPowBlock is likely to throw and startFinding is not error handled. I would catch this getPowBlock() and if it throws, just continue and try to find the merge block